src/goto-instrument/module_dependencies.txt

```diff
@@ -10,5 +10,6 @@ langapi # should go away
 linking
 pointer-analysis
 solvers
+synthesizer
```

Comment:
Unless `synthesizer` becomes a separate binary, I'd make that a subdirectory of `goto-instrument`.

Reply:
Yes, it is now a sub-directory of ```goto-instrument```.